privacy-security-explainer.md

```diff
@@ -117,10 +118,10 @@ function onSessionRafCallback(XRFrame frame) {
 }
 ```
 
-As a result, when more than one view is present, the UA must additionally ensure that:
+In addition to meeting the [`XRPose`](#xrpose) requirements, every call to `XRFrame.getViewerPose()` which will return more than one `XRView` must additionally ensure that:
 * User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent)
-* If the device supports configurable or factory-calibrated interpupillary distance that may vary from device to device, then the XRView transform data must be rounded to prevent fingerprinting. Specific precision for rounding is at the discretion of the user agent.
-* If the XRViewerPose will include multiple XRViews for displays whose positions and orientations have been configured by the user (e.g. in a CAVE) then the XRView transform data must be rounded to prevent fingerprinting.
+* If `XRView` data is affected by settings that may vary from device to device, such as static interpupillary distance or variations in screen geometry, then the XRView data must be anonymized to prevent fingerprinting. Specific approaches to this are at the discretion of the user agent.
+* If `XRView` data is affected by a user-configured interpupillary distance, then the XRView data must be anonymized to prevent profiling as well as fingerprinting. Specific approaches to this are at the discretion of the user agent.
```

Comment:
With suggestion on previous line, I believe this one can be removed as anonymization prevents fingerprinting but not profiling.

Reply:
I believe this is fixed now. Let me know if not!